spikeinterface/widgets/correlograms.py

```diff
@@ -9,6 +9,7 @@ class CrossCorrelogramsWidget(BaseWidget):
     """
     Plots spike train cross-correlograms.
     The diagonal is auto-correlogram.
+    
```

Comment:
It would be nice to not touch too much widgets because I doing a super big PR for refactoring.